decidim-core/app/views/decidim/participatory_processes/show.html.erb

```diff
@@ -73,6 +73,9 @@
   <div class="row">
     <div class="columns section view-side mediumlarge-4 mediumlarge-push-8 large-3 large-push-9">
       <%= render partial: "decidim/shared/share_modal" %>
+
+      <% embed_url = embedded_code_for(participatory_process_participatory_process_widget_path(current_participatory_process)) %>
```

Comment:
It is a bit weird for me calling `embed_url` instead of something like `embed_code` or `embed_iframe_code`.